tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/effects/debugger_effects.ts

```diff
@@ -105,6 +113,40 @@ import {
 /** @typehack */ import * as _typeHackNgrxStore from '@ngrx/store/src/models';
 /** @typehack */ import * as _typeHackNgrxEffects from '@ngrx/effects/effects';
 
+// Minimum polling interval in milliseconds.
+export const MIN_POLLING_INTERVAL_MS = 2e3;
+// Maximum polling interval in miliseconds.
+export const MAX_POLLING_INTERVAL_MS = 60e3;
+// Backoff behavior takes effect when time since the elapsed time between the
+// most recent arrival of new polling result and the most recent polling event
+// exceeds `pollingBackoffFactor * minimumPollingInterval`.
+export const POLLING_BACKOFF_FACTOR = 2;
+
+/**
+ * Get the current polling interval based on the time lapsed since
```

Comment:
code style: jsdoc should start with verb in third person tense, i.e., "Return*s* the current polling interval..."

Reply:
Done.